setup.py

```diff
@@ -141,6 +141,10 @@
     long_description=open("README.md").read(),
     long_description_content_type="text/markdown",
     zip_safe=False,
+    project_urls={
+        "Source": "https://github.com/Chia-Network/chia-blockchain/",
+        "Changelog": "https://raw.githubusercontent.com/Chia-Network/chia-blockchain/main/CHANGELOG.md",
```

Comment:
I think it would be nice if users clicking the link got the rendered markdown instead of the raw source.  Is renovate able to handle this?

Reply:
Renovate doesn't seem to care: https://github.com/strayer/chia-project-urls-test/pull/4

Although I remember reading somewhere that Renovate prefers to read Github releases if they are set in the source URL, so it actually might not really matter what is set as a changelog URL.